### ISSUE TITLE
Migrate "database" to Zotero (?)

### ISSUE THREAD

**Issue by rufuspollock** [rufuspollock/awesome-crypto-critique#44]
What we have is essentially a bibliography. The proposal here is to adopt the [Zotero reference manager](https://zotero.org) as our backend storage for the bibliographic items.

## Why

The main plus point of managing as README in git(hub) is:

* Making it easy to have moderated contribution via pull requests etc
* Quick publishing (it is just the github readme)
* Stars ⭐ (so some sense of interest / attention)

However, this is not necessarily optimal, especially as the bibliography grows. For example:

* Manual additions of new items including the key metadata
* No standard format for items (a change in standard format requires
* Painful to show the same item under multiple topics (manually copy and paste and if we correct anything needs correcting in multiple locations)
* Re-organizing of topics requires manual work
* No ability to sort by date or similar

Zotero + a renderer (static site) would seem to address a lot of this. At a minimum using zotero and very simple rendering should be able to reproduce what we have already and allow for easier addition of new material.

**Comment by rufuspollock** [comment]
@stevesong sounds good to you?

**Comment by stevesong** [comment]
Sounds good to me.  I am a little concerned about the sheer volume of resources and the need to sort the wheat from the chaff.  For a start it would be good to agree on a set of metatags for describing the resources.  More ambitious would be some consensus on the best resources for a specific topic or best introduction to issues.

**Comment by rufuspollock** [comment]
> Sounds good to me. I am a little concerned about the sheer volume of resources and the need to sort the wheat from the chaff. For a start it would be good to agree on a set of metatags for describing the resources. More ambitious would be some consensus on the best resources for a specific topic or best introduction to issues.

Big 👍 and suggest we open a separate issue for this and start working on it. I think the SCQH and esp issue tree structure could help us get a taxonomy here.

**Comment by bwiernik** [comment]
> Painful to show the same item under multiple topics (manually copy and paste and if we correct anything needs correcting in multiple locations)

This is incorrect. If you use Zotero's collections for your topics, one item can exist in many collections

**Comment by rufuspollock** [comment]
@bwiernik first, thanks for contributing and welcome 🙂

>> Painful to show the same item under multiple topics (manually copy and paste and if we correct anything needs correcting in multiple locations)

> This is incorrect. If you use Zotero's collections for your topics, one item can exist in many collections.

To clarify, the why section is about why *github readme* approach does not work and *why* we should move to zotero. I've clarified the title now ...

> In addition, with Zotero's translator framework, metadata for most types of items can be automatically created, so manual entry should rarely be necessary

👍 indeed. If you are interested the library is already operational and pretty active at https://www.zotero.org/groups/4600269/web3/library and we're looking in #52 at out how we display them in the main website.

**Comment by bwiernik** [comment]
Thanks, I'm mostly a volunteer and contributor to Zotero, so wanted to answer some questions that I saw pop up from your initial post

**Comment by danieltomasz** [comment]
@rufuspollock check this project https://github.com/whiskyechobravo/kerkoapp
there was a discussion here mentioning this web library *https://forums.zotero.org/discussion/71905/publishing-a-large-bibliography-on-the-web-with-a-faceted-search-interface
